apps/pipeline/upstream/env/aws/disable-default-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
So this will delete the secret if it exist? 
Does it fails if such secret does not exist?

Reply:
This doesn’t delete the secret it removes the secret from being installed in the first place by removing it from the yaml file when built with kustomize